setup_llvm.bzl

```diff
@@ -6,8 +6,8 @@ def setup_llvm_toolchain(name):
     mapping = {
         "linux-aarch64": {"version": "15.0.6", "triple": "aarch64-linux-gnu", "sha256": "8ca4d68cf103da8331ca3f35fe23d940c1b78fb7f0d4763c1c059e352f5d1bec"},
         "linux-x86_64": {"version": "15.0.6", "triple": "x86_64-linux-gnu-ubuntu-18.04", "sha256": "38bc7f5563642e73e69ac5626724e206d6d539fbef653541b34cae0ba9c3f036"},
-        "darwin-aarch64": {"version": "15.0.6", "triple": "arm64-apple-darwin21.0", "sha256": "32bc7b8eee3d98f72dd4e5651e6da990274ee2d28c5c19a7d8237eb817ce8d91"},
-        "darwin-arm64": {"version": "15.0.6", "triple": "arm64-apple-darwin21.0", "sha256": "32bc7b8eee3d98f72dd4e5651e6da990274ee2d28c5c19a7d8237eb817ce8d91"},
+        "darwin-aarch64": {"version": "16.0.0", "triple": "arm64-apple-darwin22.0", "sha256": "2041587b90626a4a87f0de14a5842c14c6c3374f42c8ed12726ef017416409d9"},
+        "darwin-arm64": {"version": "16.0.0", "triple": "arm64-apple-darwin22.0", "sha256": "2041587b90626a4a87f0de14a5842c14c6c3374f42c8ed12726ef017416409d9"},
```

Comment:
Weakly -1 on bumping the toolchain version for one OS and not for other OSes, because it risks creating more issues down the line. E.g. you might make some changes and those might compile locally because you're using a newer toolchain than the one being used in CI.

Historically, we've had a mismatch in 15.0.6 and 15.0.7 in some places due to the absence of x86_64 macOS artifacts for 15.0.6, and GitHub historically only supporting x86_64 macOS runners. However, the latter is no longer true.